cloud-controller-manager/do/firewall_fixture_test.go

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright 2020 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package do
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"github.com/digitalocean/godo"
+	"github.com/mitchellh/copystructure"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	ctx = context.TODO()
```

Comment:
AFAIK that's not what `context.TODO` is for. I believe we should use `context.Background` here.

Reply:
Yup you're right. Will change.